jenkins/ci.suse.de/templates/cloud-mkcloud-job-4nodes-linuxbridge-template.yaml

```diff
@@ -24,4 +24,6 @@
             tempestoptions={tempestoptions}
             mkcloudtarget=all
             label={label}
+            # Temporal fix meanwhile test repository is fixed for Ceph
```

Comment:
Might be good to add a bsc number here if there is one? So we know when it can be removed.